### PR TITLE
Updated README.md as per issue#20

### PR DESCRIPTION
**Pull request by Unnimaya6122004**
Added reflectly,watermaniac in health and fitness---natrium in finance---inkino in entertainment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +159,8 @@ No projects.
 |:-----------|:------|:-------------|
 | Flutter-Steps-Tracker      | [Link](https://github.com/TarekAlabd/Flutter-Steps-Tracker)      | It's an open source project for the steps tracking (Pedometer) built with Flutter. |
 | Just Another Workout Timer | [Link](https://github.com/blockbasti/just_another_workout_timer) | A simple timer for your workouts, built with Flutter! |
-
+| Reflectly                  | [Link](https://github.com/topics/reflectly])                     | Reflectly is a journaling app designed to help users foster a positive mindset and improve their mental well-being through daily reflections |
```
Comment (ashitaprasad) [inline README.md:162]: Add the link for the github repo of the project here

Comment (Unnimaya6122004) [reply README.md:162]: Where shouldi add..162th line?

Comment (Unnimaya6122004) [reply README.md:162]: And also the link u are talking about is ...can u specify pls...😇

Comment (ashitaprasad) [reply README.md:162]: Yes, add the github repo of the flutter open source app. The current link that you have added is incorrect.

**Comment by Unnimaya6122004** [comment]
i have entered the valid now...can u check it now..

**Comment by Unnimaya6122004** [comment]
is everything okay now...

**Comment by ashitaprasad** [comment]
Yes

**Comment by Unnimaya6122004** [comment]
what should i do next mam...

**Comment by ashitaprasad** [comment]
Now if you want to add more projects, raise a new issue and create a new PR for it accordingly. 
For this PR, nothing more needs to be done from your side as it is already merged.

**Pull request merged**